packages/data-api/src/SqlQuery.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
Minor: it looks like Github isn't smart enough to know this is a renamed file, it sees it as an Added file and a Deleted file. To make it easier to review, maybe in the future we should do this in two separate commits, one for the rename only, and the other to make changes?

Reply:
Ah good point. Tbh I always do my reviews as bulk rather than by commit, so I don't often think about the reviewer's perspective like that, but I'll try to be more mindful from now on 👍

Reply:
Yeah me too, but in this case even in full-change view it shows SqlQuery twice, so commit by commit might be easier. We don't typically rename files like this so not a biggie.